src/test/rmc/Iterator/try_fold.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// cbmc-flags: --unwind 20
```

Comment:
Crazy that we need an unwind of 20 for this example

Reply:
Also non-intuitive for users, which I think is a real issue. We should get this straight when we look into unwinding.

Reply:
That was just a random number. I can bring it down if you are worried about that.